includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -0,0 +1,276 @@
+		}
+
+		return $this->wikiLocalNavigation;
+	}
+
+	public function getExploreMenu(): array {
+		if ( $this->exploreMenu === null ) {
+			$wgEnableCommunityPageExt = WikiFactory::getVarValueByName( 'wgEnableCommunityPageExt', $this->productInstanceId );
+			$wgEnableForumExt = WikiFactory::getVarValueByName( 'wgEnableForumExt', $this->productInstanceId );
+			$wgEnableDiscussions = WikiFactory::getVarValueByName( 'wgEnableDiscussions', $this->productInstanceId );
+			$wgEnableSpecialVideosExt = WikiFactory::getVarValueByName( 'wgEnableSpecialVideosExt', $this->productInstanceId );
+
+			$exploreItems = [
+				[
+					'title' => 'WikiActivity',
+					'key' => 'community-header-wiki-activity',
+					'tracking' => 'explore-activity',
+					'include' => true,
+				],
+				[
+					'title' => 'Random',
+					'key' => 'community-header-random-page',
+					'tracking' => 'explore-random',
+					'include' => true,
+				],
+				[
+					'title' => 'Community',
+					'key' => 'community-header-community',
+					'tracking' => 'explore-community',
+					'include' => !empty( $wgEnableCommunityPageExt ),
+				],
+				[
+					'title' => 'Videos',
+					'key' => 'community-header-videos',
+					'tracking' => 'explore-videos',
+					'include' => !empty( $wgEnableSpecialVideosExt ),
+				],
+				[
+					'title' => 'Images',
+					'key' => 'community-header-images',
+					'tracking' => 'explore-images',
+					'include' => true,
+				],
+				[
+					'title' => 'Forum',
+					'key' => 'community-header-forum',
+					'tracking' => 'explore-forum',
+					'include' => !empty( $wgEnableForumExt ) && !empty( $wgEnableDiscussions ),
+				],
+			];
+
+			$this->exploreMenu = [
+				'type' => 'dropdown',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'community-header-explore',
+				],
+				'image-data' => [
+					'type' => 'wds-svg',
+					'name' => 'wds-icons-explore-tiny'
+				],
+				'items' => array_map(
+					function ( $item ) {
+						return [
+							'type' => 'link-text',
+							'title' => [
+								'type' => 'translatable-text',
+								'key' => $item['key'],
+							],
+							'href' => GlobalTitle::newFromText( $item['title'], NS_SPECIAL, $this->productInstanceId )
+								->getFullURL(),
+							'tracking_label' => $item['tracking']
+						];
+					},
+					array_values(
+						array_filter(
+							$exploreItems,
+							function ( $item ) {
+								return $item['include'];
+							}
+						)
+					)
+				)
+			];
+		}
+
+		return $this->exploreMenu;
+	}
+
+	public function getDiscussLinkData(): array {
+		if ( $this->discussLinkData === null ) {
+			$wgEnableForumExt = WikiFactory::getVarValueByName( 'wgEnableForumExt', $this->productInstanceId );
+			$wgEnableDiscussions = WikiFactory::getVarValueByName( 'wgEnableDiscussions', $this->productInstanceId );
+
+			$url = '';
+			$key = '';
+			$tracking = '';
+
+			if ( !empty( $wgEnableDiscussions ) ) {
+				$url = $host = WikiFactory::getHostById( $this->productInstanceId ) . '/d/f';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$host" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)